docs/components/Heading/Heading.stories.mdx

```diff
@@ -0,0 +1,68 @@
+| `<Heading level={6} />` | Used to group contents after a `level={5}` Heading or to group small lists of content.                                                                                                                                                  |
+
+  </Tab>
+
+  <Tab label="Mobile">
+
+| Level                              | Use case                                                                                                                                                           |
+| ---------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| `<Heading level={"title"} />`      | Should explain the main subject of the page. There should only be one `title` Heading on a page.                                                                   |
+| `<Heading level={"subtitle"} />`   | Used to categorize large groups of content.                                                                                                                        |
+| `<Heading level={"heading"} />`    | Used to group content and forms on a single topic. Like `title` and `subtitle`, `heading` uses [Jobber Pro](../?path=/docs/design-typography--page#font-families). |
+| `<Heading level={"subHeading"} />` | Used to group small lists of content, `subHeading` uses the [Inter](../?path=/docs/design-typography--page#base) font.                                             |
+
+  </Tab>
+</Tabs>
+
+## Content guidelines
+
+The only content that should be used in a Heading is text. Use sentence-case for
+headings. The exception is `Heading 6` in the web Headings, which is capitalized
+as a stylistic reference to the
+[eyebrow typographic pattern.](https://app.uxcel.com/lessons/basics-ii-588#eyebrow-headline-8095)
+
+| ✅ Do                       | ❌ Don't                    |
+| --------------------------- | --------------------------- |
+| Job #21 for Nathaniel Lewis | Job #21 For Nathaniel Lewis |
+| Save job and...             | Save Job And...             |
+| Push notification settings  | Push Notification Settings  |
+
+## Related components
+
+For introductory text after a page title, or paragraph text for body copy, use
+[Text](../?path=/docs/components-text-and-typography-text-docs--page).
+
+## Platform considerations
+
+There are a few caveats around copying text on Android and iOS that you can read
+under the
+[Typography](../?path=/docs/components-text-and-typography-typography-docs--page#platform-considerations)
+documentation.
+
+## Accessibility
+
+The Heading component has the accessibilityRole of 'Header' and will be
+announced to assistive technology as "Header: {text content}".
```

Comment:
Do we need to explain this?

Also the `{text content}` string would break on storybook 7 so maybe use [] or just ay text content really.

Reply:
I took it from the mobile docs. I could see that most people might assume this already, but I'm not sure it's taking anything away from the documentation by being there. It's another way of saying, use Heading instead of big bold text to give context for assistive technology users.

Reply:
IMO, it's one of those implied things where if we start explaining them it will be a thing we have to explain everywhere. Or in other words, assume everything is working unless otherwise stated.

Reply:
Fair enough. I removed that section.